### DISCUSSION TITLE
More devices

### DISCUSSION THREAD

**Discussion by cozyop** [bookiza/--inch#13] [General]
Hey this is a great project!
I tested on a few devices and it was working well on Desktop, but different numbers on my phone. 
How do you get the numbers for the devices? Do you go by testing on each device or from the known dpi?

**Comment by marvindanig** [comment]
Appreciate your comments. The project started with testing for the dpi pairings on the devices my friends and I owned. So yeah, it has been manually done using a plastic ruler I snagged at the dollar store. 😬 

![scale](https://user-images.githubusercontent.com/5659924/210415250-1c98f847-147f-49a7-ab89-e20c6e2cb9da.jpg)
